### PR TITLE
Improve livechat widget elements spacing

### PR DESCRIPTION
**Pull request by sampaiodiego**
@RocketChat/core 

Before:
image

After:
image

### CODE REVIEW

File: packages/rocketchat-livechat/app/client/stylesheets/main.less [hunk 1]
```diff
@@ -140,23 +140,25 @@ input:focus {
...
-			padding-top: 3px;
-			height: 30px;
 			svg {
 				cursor: pointer;
 				fill: currentColor;
 				width: 14px;
-				margin: 0 2px;
+				margin: 0 5px;
+				vertical-align: middle;
+
```
Comment (karlprieb) [inline packages/rocketchat-livechat/app/client/stylesheets/main.less]: Can you remove this line?

Comment (sampaiodiego) [inline packages/rocketchat-livechat/app/client/stylesheets/main.less]: we cannot have **any** empty lines? really?

Comment (marceloschmidt) [inline packages/rocketchat-livechat/app/client/stylesheets/main.less]: sometimes empty lines improve readability... could we allow them?

Comment (sampaiodiego) [inline packages/rocketchat-livechat/app/client/stylesheets/main.less]: exactly.. where this rules came from? I'm looking for general standards, like [this one](https://github.com/stylelint/stylelint-config-standard) which is generated from google, airbnb and others stantards..

Comment (karlprieb) [inline packages/rocketchat-livechat/app/client/stylesheets/main.less]: Most of this rules came from @engelgabriel

**Pull request merged**